config/locales/simple_form.en.yml

```diff
@@ -20,3 +20,14 @@ en:
         platform_provider_name:
           twilio: "Twilio"
           somleng: "Somleng"
+  helpers:
+    submit:
+      sensor:
+        create: "Create Sensor"
+        update: "Save"
+      sensor_rule:
+        create: "Create Sensor Rule"
+        update: "Save"
+      callout:
+        create: "Create Callout"
+        update: "Update Callout"
```

Comment:
Translate this to `Save` for consistency

Reply:
Delete all of these translations.